uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Upgrader/Steps/1031.php

```diff
@@ -8,11 +8,35 @@
    * @return bool
    */
   public function upgrade_1031() {
+    $this->up1031_updateManagedEntitiesTable();
     $this->up1031_disableAndUninstallRecruitment();
 
     return TRUE;
   }
 
+  /**
+   * Making sure that the entity_id of CaseType and Managed are equal
+   */
+  private function up1031_updateManagedEntitiesTable() {
+    $caseType = civicrm_api3('CaseType', 'get', [
+      'return' => ['id'],
+      'name' => 'Application',
+    ]);
+    if($caseType['count']===0){
```

Comment:
You're missing some spaces here and it would look clearer with some empty lines around the `if` block